2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+Title: Funding Freedom Society - Idee zur Finanzierung freier Softwareprojekte
+Date: 2018-03-04 9:00
+Category: Artikel
+Tags: Freie Software, Gesellschaft
+Slug: FFS-wp
+Author: Carsten
+Summary: Finanzierungsmodell zur Steigerung von Qualität und Verbreitung von FLOSS durch
+Kombination existierender Geschäftsmodelle (Spenden, Merchandise, Bounties, Funding-Organisationen.)
+
+---
+
+# Funding Freedom Society
+
+Dieser Text beschreibt eine Idee zur Steigerung von Qualität und Verbreitung von freier quelloffener Software ([FLOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software)) und freien dezentralen Web-Diensten.
+Er geht nicht darauf ein, *warum* diese Ziele erstrebenswert sind.
+Er ist eine stark inspiriert durch [Colin Percivals Plan für Open Source Software Maintainer](http://www.daemonology.net/blog/2017-05-11-plan-for-foss-maintainers.html).
+
+## Bestandsaufnahme
+
+
+Ausgangslage: Mit zunehmender Digitalisierung wird Software privat und beruflich immer wichtiger.
+Freie quelloffene Software für Endbenutzer*innen ist XX Jahre nach Etablierung des Begriffes OpenSource immer noch (oder mehr denn je) ein Nischenprodukt.
```

Comment:
Ein Nischenprodukt auf Endbenutzergeräten. Denn Serversoftware ist zu garnicth so kleinen Teilen FLOSS, siehe Apache, Nginx, Postfix, Dovecot, Exim, Qmail, PowerDNS, BIND, Bird, Quagga, diverse VPN/Tunnel-Dinge….

Reply:
Stimmt. Aber es geht hier in erster Linie um Desktop-Software. Unterstützt unser Blog Fußnoten? oder aufklappbare Boxen? Dann könnte man die Server-Situtation schildern, ohne die Hauptargumentation zu sehr zu unterbrechen. Alternativ könnte man diesen Punkt ganz unten nochmal aufgreifen, wo es kurz um die Finanzierung von Diensten geht.

Reply:
Ist so wahrscheinlich klar genug, weil "Endbenutzer*innen".